test/core_language/test/bit_string_test.gleam

```diff
@@ -9,11 +9,26 @@ fn integer_fn() {
 // Valid values
 
 pub fn function_as_value_test() {
-  let <<a>> = <<integer_fn():integer>>
+  let <<a>> = <<integer_fn():int>>
 
   should.equal(a, 1)
 }
 
+pub fn integer_to_binary_test() {
+  let <<a, rest:binary>> = <<1, 17, 42:16>>
+
+  should.equal(rest, <<17, 0, 42>>)
+}
+
+// Sizes
+
+pub fn size_variable_from_match_test() {
+  let <<name_size:8, name:binary-size(name_size), " the ":utf8, species:binary>> = <<5, "Frank the Walrus":utf8>>
```

Comment:
Love this example!

Has the formatter been run on this file? Seems like a long line.

Reply:
Ah, it has not. Let me fix that!

Reply:
Turns out we hadn't formatted the core_language project for a while (ever?) so there's some bonus clean up in a new commit